tests/v2/validation/provisioning/rke1/provisioning_node_driver_test.go

```diff
@@ -129,13 +132,16 @@ func (r *RKE1NodeDriverProvisioningTestSuite) TestProvisioningRKE1Cluster() {
 				name = tt.name + providerName + " Kubernetes version: " + kubeVersion
 				for _, cni := range r.cnis {
 					name += " cni: " + cni
-					scaleName += name + " scaling"
+					scaleName = name + " scaling"
 					r.Run(name, func() {
-						r.testProvisioningRKE1Cluster(client, provider, tt.nodeRoles, kubeVersion, cni)
+						cluster, err := r.testProvisioningRKE1Cluster(client, provider, tt.nodeRoles, kubeVersion, cni)
+						require.NoError(r.T(), err)
+
+						tt.cluster = cluster
```

Comment:
Couldn't this just be a variable? What's the need for it to be a test parameter?

Reply:
When I was testing regex to split scaling from non-scaling, I found I had to have that for it to work. I'll try to remove it and see if I can get it to work.

Reply:
Really? In that case I think it makes more sense to have it as a suite parameter like how it is in line 112. But make sure to make it nil after running the tests ie. after the r.Runs() r.cluster = nil.

Reply:
Suite parameter, you mean just leave it as and make it nil after the r.Runs() or define `cluster *management.Cluster` in the beginning of the file? Sorry, thrown off by the wording of suite parameter in this context.

Reply:
Never mind, I was confusing myself, got it.

Reply:
Oh sorry I misread the file. The suite parameter are the ones starting from line 26. Define the cluster there, and then reset after the r.Runs(). I think that will work. Having it as test parameter ie tt.cluster, is a bit confusing to see.